src/Model/Survey/Survey.cs

```diff
@@ -1,28 +1,154 @@
-namespace Survey;
+using System;
+namespace Model.Survey;
```

Comment:
namespace should before any imports
```suggestion
namespace Model.Survey;
using System;
```